web/app/components/user-info/user-info.tsx

```diff
@@ -0,0 +1,88 @@
+/** @jsx h */
+import { h, Component, RenderableProps } from 'preact';
+import b from 'bem-react-helper';
+import { connect } from 'preact-redux';
+
+import { StoreState, StoreDispatch } from '@app/store';
+import { Comment } from '@app/common/types';
+import { fetchInfo } from '@app/store/user-info/actions';
+import { userInfo } from '@app/common/user-info-settings';
+
+import LastCommentsList from './last-comments-list';
+import { AvatarIcon } from '../avatar-icon';
+
+interface Props {
+  comments: Comment[] | null;
+  fetchInfo: () => Promise<Comment[] | null>;
+}
+
+interface State {
+  isLoading: boolean;
+  error: string | null;
+}
+
+class UserInfo extends Component<Props, State> {
+  state = { isLoading: true, error: null };
+
+  async componentWillMount(): Promise<void> {
```

Comment:
why async here?

Reply:
Ouch, remained of something while refactoring, thanks.